### PR TITLE
Upgrade powsybl versions and refactor packages organization

### PR DESCRIPTION
**Pull request by TheMaskedTurtle**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Adaptation of code for release v2.0.0. Upgrade PowSyBl and checkstyle version. Clean packages for stable release.

### CODE REVIEW

File: data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/JsonImport.java [hunk 1]
```diff
@@ -32,16 +28,7 @@
  */
 @AutoService(CracImporter.class)
 public class JsonImport implements CracImporter {
-
-    private static final String CRAC_FILE_SCHEMA_JSON = "/CracSchema.json";
```
Comment (terriervik) [inline data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/JsonImport.java]: You can maybe delete this file if you delete its usages.

File: flowbased-computation/flowbased-computation-impl/pom.xml [hunk 2]
```diff
@@ -37,6 +37,11 @@
         </dependency>
 
         <!-- Test dependencies -->
+        <dependency>
+            <groupId>com.powsybl</groupId>
```
Comment (terriervik) [inline flowbased-computation/flowbased-computation-impl/pom.xml]: Please respect the alphabet.

File: data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/JsonImport.java (hunk 1 above)
Comment (TheMaskedTurtle) [reply data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/JsonImport.java]: ok

File: flowbased-computation/flowbased-computation-impl/pom.xml (hunk 2 above)
Comment (TheMaskedTurtle) [reply flowbased-computation/flowbased-computation-impl/pom.xml]: ok

**Pull request merged**